### PR TITLE
Fix Rotation to be randomized

### PR DESCRIPTION
**Pull request by Ilya2000Klimov**
## Description
- [X] fix the `Rotate` class in `dataset_aug.py` to make the rotations random

## Files Changed
- `dataset_aug.py`
- `train_val_split.py`

## Screenshot
Figure_1

### CODE REVIEW

File: search_simclr/simclr/dataloader/dataset_aug.py [hunk 1]
```diff
@@ -348,16 +349,16 @@ def __init__(self,
...
+            # transforms.RandomApply([H_Flip()], p=0.5),
+            # transforms.RandomApply([V_Flip()], p=0.5),
+            # transforms.RandomApply([P_Flip()], p=0.5), 
+            transforms.RandomApply([Rotate(rotate)], p=1),
+            # transforms.RandomApply([Brighten(brighten)], p=0.5),
+            # transforms.RandomApply([Translate(translate)], p=0.5),
+            # transforms.RandomApply([Zoom(zoom)], p=0.5),
+            # transforms.RandomApply([Cutout(cutout_holes, cutout_size)], p=0.5),
+            # transforms.RandomApply([Blur(blur)], p=0.5),
+            # transforms.RandomApply([AddNoise(noise_mean, noise_std)], p=0.5),
```
Comment (ReOp14) [inline search_simclr/simclr/dataloader/dataset_aug.py:361]: Transformations should be left uncommented 😄

**Review by ReOp14 — CHANGES_REQUESTED** [review]
Looking good!

File: search_simclr/simclr/dataloader/dataset_aug.py (hunk 1 above)
Comment (Ilya2000Klimov) [reply search_simclr/simclr/dataloader/dataset_aug.py:361]: Thank you for noticing

**Comment by nadia-eecs** [comment]
- Should update transformations in `datamodule.py` in addition to the script.
- Looks good :+1:

**Review by ReOp14 — APPROVED** [review]

**Pull request merged**